src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -37,8 +46,13 @@
     public static final String ERROR_STYLE_CLASS = "error";
     private static final String FXML = "CommandBox.fxml";
     private static final int TIME_SINCE_TYPING = 300;
```

Comment:
Think I missed this in a previous commit. Perhaps be more specific in variable naming conventions? `MILLISECONDS_SINCE_TYPING`, `MILLISECONDS_PAUSE_BEFORE_PRESSING_CTRL`, ...

Reply:
@jo-lyn